src/zap/zapheaders.cpp

```diff
@@ -282,6 +282,7 @@ void ZapDebugDirectory::SaveOriginalDebugDirectoryEntry(ZapWriter *pZapWriter)
 
 void ZapDebugDirectory::SaveNGenDebugDirectoryEntry(ZapWriter *pZapWriter)
 {
+#if !defined(NO_NGENPDB)
```

Comment:
I'm not sure I like the name of this ifdef  NO_NGENPDB

It appears to  just mean all UNIX platforms. 
If there isn't a realistic plan to make PDBs for UNIX it might be better just to use "ifdef PLATFORM_UNIX"
As it make is more obvious from reading the source code that this is something that isn't implemented for UNIX platforms.   


It is defined here in tools\crossgen\CMakeLists.txt:17:

if(CLR_CMAKE_PLATFORM_UNIX)
  add_compile_options(-fPIE)
  add_definitions(-DNO_NGENPDB)
endif(CLR_CMAKE_PLATFORM_UNIX)